raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -2182,6 +2182,46 @@ public void testHandleCommitCallbackFiresInCandidateState() throws Exception {
         assertEquals(OptionalInt.empty(), secondListener.currentClaimedEpoch());
     }
 
+    @Test
+    public void testObserverFetchWithNoLocalId() throws Exception {
+        // When no `localId` is defined, the client will behave as an observer.
```

Comment:
Wondering whether we could copy this comment to the raft client localId field.

Reply:
Makes sense. I will move it.